packages/celeste-vue/src/components/textarea/textarea.vue

```diff
@@ -0,0 +1,158 @@
+<script setup lang="ts">
+import type { TextareaHTMLAttributes } from 'vue';
+import clsx from 'clsx';
+
+const props = withDefaults(defineProps<TextAreaProps>(), {
+  disabled: false,
+  placeholder: 'Placeholder',
+  showCount: true,
+});
+
+const modelValue = defineModel<string>();
+</script>
+
+<script lang="ts">
+export interface TextAreaProps {
+  class?: TextareaHTMLAttributes['class'];
+  disabled?: boolean;
+  placeholder: string;
+  error?: boolean;
```

Comment:
```suggestion
  placeholder: string;
```

We can just take out those attributes, you can target them directly in CSS, and it'll save us a bunch of extra lines

```css
&[disabled] {}
&[error] {}
```

Reply:
laziness :caught: